modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -20,45 +20,156 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 .Procedure
 
-. Create the `Secret` CR and the `BareMetalHost` CR.
-
-.. Save the following YAML in the `bmh.yaml` file:
+. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
```

Comment:
```suggestion
. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
```

![Screenshot from 2025-01-07 11-58-39](https://github.com/user-attachments/assets/bfc7e94a-e8b1-4a1b-8fa9-096bf089d7b9)

Consider: 

Depending if you use either a static configuration or a DHCP server, choose one of the following example `bmh.yaml` files and configure it to your needs by replacing...